### DISCUSSION TITLE
Develco Smoke successful on one controller, fails on another

### DISCUSSION THREAD

**Discussion by kolaf** [Koenkk/zigbee2mqtt#15346] [General]
I'm running two parallel Z2M instances, one controlling a Sonoff ZBDongle-P and the other one controlling a SMLIGHT SLZB-06.  The Z2M version is the latest as of a few days ago, the Sonoff dongle is running the 20221102 firmware, and the SLZB-06 is running stock as I got it (revision 20220219). I have several Develco Smoke alarms running firmware version 4.0.2 that successfully get connected and recognised by the Sonoff controller, but fail to be recognised by the SLZB-06, it only says unsupported.  Of course, other devices power successfully to the SLZB-06.  Do you have any suggestions?

SLZB-06
![image](https://user-images.githubusercontent.com/1871212/205483252-3cc71696-acc6-4e54-baaa-88b48f9601f7.png)
![image](https://user-images.githubusercontent.com/1871212/205483302-ffc39cca-aa94-4509-a211-1956dedc6492.png)

Sonoff
![image](https://user-images.githubusercontent.com/1871212/205483286-caf66cca-c797-4680-8501-298bcc78fc5b.png)
![image](https://user-images.githubusercontent.com/1871212/205483312-7be3871c-4eaa-457c-8804-5c5c30ca3ec3.png)

Log from failed SLZB-06 pairing:
`
Zigbee2MQTT:info  2022-12-04 10:05:12: Device '0x0015bc003100cd5b' joined
Zigbee2MQTT:info  2022-12-04 10:05:12: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x0015bc003100cd5b","ieee_address":"0x0015bc003100cd5b"},"type":"device_joined"}'
Zigbee2MQTT:info  2022-12-04 10:05:12: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":{"friendly_name":"0x0015bc003100cd5b"},"type":"device_connected"}'
Zigbee2MQTT:info  2022-12-04 10:05:12: MQTT publish: topic 'zigbee2mqtt/0x0015bc003100cd5b', payload '{"last_seen":"2022-12-04T09:05:12.939Z"}'
Zigbee2MQTT:info  2022-12-04 10:05:12: Starting interview of '0x0015bc003100cd5b'
Zigbee2MQTT:info  2022-12-04 10:05:12: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x0015bc003100cd5b","ieee_address":"0x0015bc003100cd5b","status":"started"},"type":"device_interview"}'
Zigbee2MQTT:info  2022-12-04 10:05:12: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_started","meta":{"friendly_name":"0x0015bc003100cd5b"},"type":"pairing"}'
Zigbee2MQTT:info  2022-12-04 10:05:13: MQTT publish: topic 'zigbee2mqtt/0x0015bc003100cd5b', payload '{"last_seen":"2022-12-04T09:05:13.318Z"}'
Zigbee2MQTT:info  2022-12-04 10:05:13: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x0015bc003100cd5b","ieee_address":"0x0015bc003100cd5b"},"type":"device_announce"}'
Zigbee2MQTT:info  2022-12-04 10:05:13: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"0x0015bc003100cd5b"},"type":"device_announced"}'
Zigbee2MQTT:info  2022-12-04 10:05:44: MQTT publish: topic 'zigbee2mqtt/0x0015bc003100cd5b', payload '{"last_seen":"2022-12-04T09:05:44.638Z"}'
Zigbee2MQTT:info  2022-12-04 10:05:45: MQTT publish: topic 'zigbee2mqtt/0x0015bc003100cd5b', payload '{"last_seen":"2022-12-04T09:05:45.032Z"}'
Zigbee2MQTT:info  2022-12-04 10:05:45: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x0015bc003100cd5b","ieee_address":"0x0015bc003100cd5b"},"type":"device_announce"}'
Zigbee2MQTT:info  2022-12-04 10:05:45: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"announce","meta":{"friendly_name":"0x0015bc003100cd5b"},"type":"device_announced"}'
Zigbee2MQTT:error 2022-12-04 10:07:35: Failed to interview '0x0015bc003100cd5b', device has not successfully been paired
Zigbee2MQTT:info  2022-12-04 10:07:35: MQTT publish: topic 'zigbee2mqtt/bridge/event', payload '{"data":{"friendly_name":"0x0015bc003100cd5b","ieee_address":"0x0015bc003100cd5b","status":"failed"},"type":"device_interview"}'
Zigbee2MQTT:info  2022-12-04 10:07:35: MQTT publish: topic 'zigbee2mqtt/bridge/log', payload '{"message":"interview_failed","meta":{"friendly_name":"0x0015bc003100cd5b"},"type":"pairing"}'
`

**Comment by sergge1** [comment]
@kolaf probablyDevelco Smoke alarms Z2M support was added in dev branch v 20221102. So update another coordinator with the same firmware to have the same results
